modules/icgc-argo-workflows/prep/metrics/resources/usr/bin/prealn.py

```diff
@@ -209,22 +178,6 @@ def main():
     with open("%s.argo_metrics.json" % (args.sampleId), 'w') as f:
```

Comment:
Same as above. 
Change file pattern to contain more details to avoid the chance of file being overwritten when the same folder contains different types of metric files for the same sample.
For example: "%s.prealn_metrics.json"